ui/lib/apps/InstanceProfiling/pages/Detail.tsx

```diff
@@ -127,7 +127,7 @@ export default function Page() {
       }
 
       // make both generated graph(svg) file and protobuf file viewable online
-      let profileURL = `${client.getBasePath()}/profiling/single/view?token=${token}`
+      let profileURL = `${client.getApiBaseURL()}/profiling/single/view?token=${token}`
```

Comment:
I recommend to call it `getAbsoluteBasePath`, to ensure the "absolute" behavior.

Reply:
Make sense!  We may have the same requirement in the future, so we can support the `getAbsoluteBasePath` for future use in the next time. How about importing profiling file via relative URL instead of absolute path, see relative [PR](https://github.com/pingcap/tidb-dashboard/pull/1082).